tests/test_compile_models.py

```diff
@@ -1,6 +1,6 @@
 import boto3
 from botocore.stub import Stubber
-from enclave_mgmt import compile_models
+from enclave_mgmt import compile_models, data_collection
```

Comment:
Would it help with clarity if we were to create a `test_data_collection.py` and move the corresponding tests there?